scripts/new_project.kts

```diff
@@ -225,6 +240,16 @@ object NewProject {
         println("\n${message}\n")
         System.exit(exitCode)
     }
+
+    private fun String.uppercaseEveryFirstCharacter(): String {
+        return this.split(SPACE_SEPARATOR).joinToString(separator = SPACE_SEPARATOR) { string ->
+            string.replaceFirstChar { it.uppercase() }
+        }
+    }
```

Comment:
```suggestion
        return this.replaceFirstChar { it.titlecase(Locale.getDefault()) }, "java.util.Locale")
    }
```
```suggestion
    private fun String.toTitleCase(): String {
        return this.replaceFirstChar { it.titlecase(Locale.getDefault()) }
    }
```

Reply:
Let's say the argument for `app-name` is **"my project"**, the end result should be **"My Project"**

If I'm not mistaken, the end result will be **"My project"** in this case 💭

Reply:
@Tuubz Title case is `My Project`, not the `Capitalize` one. 🙏🏻

Reply:
@Tuubz Oh, you're correct. But I do recall that there is a function to do this. let me find out.

Reply:
@kaungkhantsoe @Tuubz My bad, the [titlecase](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/titlecase.html) works only with the CharArray. And it comes out of the complexity anyway. 

BTW, for the current implementation, this function will not work with expected if the app name contains more than one space. For example, `my  project` -> it will return `My  Project`.
So I'd recommend to do this:
```suggestion
    private fun String.uppercaseEveryFirstCharacter(): String {
        return this.trim().split("\\s+".toRegex())
         .joinToString(" ") { string ->
            string.replaceFirstChar { it.uppercase() }
         }
    }
```

Sorry for the formatting. 🙏🏻

Reply:
This is the result when testing with the current implementation:
![Screen Shot 2022-07-11 at 12 13 13 PM](https://user-images.githubusercontent.com/12026942/178193327-3610ea4e-9133-441d-88c1-02811d499ea4.png)

This is the result of my suggestion:
![Screen Shot 2022-07-11 at 12 14 15 PM](https://user-images.githubusercontent.com/12026942/178193431-b2284880-9bcc-449a-91a3-f03433c24e30.png)

Reply:
Updated @lydiasama 🙌